examples/getstarted/package.json

```
@@ -35,7 +35,7 @@
     "uuid": "getstarted"
   },
   "engines": {
-    "node": ">=12.22.0 <=16.x.x",
+    "node": ">=14.19.1 <=16.x.x",
```

Shouldn't this be 
```suggestion
    "node": ">=14.x.x <=16.x.x",
```

Or is there any reason why we don't support all node@14 versions?

---

I believe there was some security risks that were patched but can't find the news release.

---

That, also the 14.19.1 is the stable 14 version and includes bug fixes & new features.